sdk/daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/DamlScriptLtsTestRunner.scala

```diff
@@ -36,12 +36,14 @@ abstract class DamlScriptLtsTestRunner extends GenericDamlScriptTestRunner {
     "pick up all scripts and returns somewhat sensible outputs for daml-script-lts features" in
       assertDamlScriptRunnerResult(
         trySubmitTestDarPath,
+        // scripts are executed in alphabetical order
         f"""DamlScriptLtsTrySubmit:authorizationError SUCCESS
            |DamlScriptLtsTrySubmit:contractKeyNotFound SUCCESS
            |DamlScriptLtsTrySubmit:contractNotActive ${expectedContractNotActiveResponse}
            |DamlScriptLtsTrySubmit:createEmptyContractKeyMaintainers SUCCESS
            |DamlScriptLtsTrySubmit:duplicateContractKey SUCCESS
            |DamlScriptLtsTrySubmit:fetchEmptyContractKeyMaintainers SUCCESS
+           |DamlScriptLtsTrySubmit:prefetchContractKeys SUCCESS
```

Comment:
This is not really testing `trySubmit`, but this was the place where most other submission option builders where tested. LMK ff there's better place for such tests.

Reply:
I think this is fine, this test has extended a bit outside of TrySubmit. You could perhaps rename the dar/module to match

Reply:
I removed `Try` from the module and dar name.